src/proto/console/text/output.rs

```diff
@@ -265,7 +265,6 @@ impl_proto! {
 /// The first 8 colors can also be used as background colors.
 #[allow(missing_docs)]
 #[derive(Debug, Copy, Clone)]
-#[repr(u8)]
```

Comment:
This enum is not used in FFI and therefore does not need a special representation.